Makefile

```diff
@@ -3,16 +3,20 @@ PROJECT_NAME := Pulumi Amazon Web Services (AWS) EKS Components
 VERSION         := $(shell pulumictl get version)
 TESTPARALLELISM := 12
 
+PACK			:= eks
```

Comment:
Nit: odd spacing

Reply:
ah I see it's tabs not spaces 🙄 - I will fix that up in the next set of work required here :)